site/components/NewsletterForm.tsx

```diff
@@ -0,0 +1,147 @@
+import Script from 'next/script';
+
+export default function NewsletterForm() {
```

Comment:
I wish we could make this simpler like classic jamstack forms stuff like getbasin or formspree. I wonder if there is a way to just use the sendinblue endpoint and none of their styling. Something to think about in future.

Reply:
I think we could do that. The endpoint seems to be quite simple. 

![ss-2023-04-19-08:45:51](https://user-images.githubusercontent.com/16550934/233065164-09fd4c7f-eeba-4a24-b5d1-1526b154cd76.png)

E.g:
```bash
curl 'https://0613d040.sibforms.com/serve/MUIEAGd31Nknuf7_fodoNJ3t0B71KWpbFfnzgk_VewvONuLQG8JO3qOotew23kQT3HpoJQUG_vzcjOTjn29B6GpXxPbSml_XWwHgG2mWq-jhrjfqpHCcPoOY_ge-rN2vDFWYZ80l242DTYGDRRWtTusdAYIk2oyf-nhJyOqQrUzTnXlAlKc7SxWgynSQ1GHr3jU5s57h6986IoK4?isAjax=1' \
  -H 'authority: 0613d040.sibforms.com' \
  -H 'accept: */*' \
  -H 'accept-language: en-US,en;q=0.9' \
  -H 'cache-control: no-cache' \
  -H 'content-type: multipart/form-data; boundary=----WebKitFormBoundaryDr7XIjKtTBPpo6kY' \
  -H 'origin: http://localhost:3000' \
  -H 'pragma: no-cache' \
  -H 'referer: http://localhost:3000/' \
  -H 'sec-ch-ua: "Chromium";v="110", "Not A(Brand";v="24", "Google Chrome";v="110"' \
  -H 'sec-ch-ua-mobile: ?0' \
  -H 'sec-ch-ua-platform: "Linux"' \
  -H 'sec-fetch-dest: empty' \
  -H 'sec-fetch-mode: cors' \
  -H 'sec-fetch-site: cross-site' \
  -H 'user-agent: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/110.0.0.0 Safari/537.36' \
  --data-raw $'------WebKitFormBoundaryDr7XIjKtTBPpo6kY\r\nContent-Disposition: form-data; name="NAME"\r\n\r\njoão\r\n------WebKitFormBoundaryDr7XIjKtTBPpo6kY\r\nContent-Disposition: form-data; name="EMAIL"\r\n\r\njoao.demenech@datopian.com\r\n------WebKitFormBoundaryDr7XIjKtTBPpo6kY\r\nContent-Disposition: form-data; name="form-name"\r\n\r\nget-updates\r\n------WebKitFormBoundaryDr7XIjKtTBPpo6kY\r\nContent-Disposition: form-data; name="email_address_check"\r\n\r\n\r\n------WebKitFormBoundaryDr7XIjKtTBPpo6kY\r\nContent-Disposition: form-data; name="locale"\r\n\r\nen\r\n------WebKitFormBoundaryDr7XIjKtTBPpo6kY--\r\n' \
  --compressed
```